includes/templates/bootstrap/css/stylesheet_zca_colors.php

```diff
@@ -11,6 +11,9 @@
 .form-control::placeholder {color: <?php echo ZCA_BODY_PLACEHOLDER; ?>;}
 .required-info, .alert {color: <?php echo ZCA_BODY_PLACEHOLDER; ?>;}
 
+/* Nav Bar Active Links */
+.activeLink {color: <?php echo ZCA_ACTIVE_LINK_COLOR; ?> !important;background-color: <?php echo ZCA_ACTIVE_LINK_BACKGROUND_COLOR; ?> !important;border: 2px solid <?php echo ZCA_ACTIVE_LINK_BORDER_COLOR; ?> !important;}
```

Comment:
This is _supposed_ to provide colors, not dimensions.  Please remove the 2px border sizing; the majority of the templates (and the base bootstrap) use rems, anyway.